docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc

```diff
@@ -23,59 +23,63 @@ The minimum supported Mimir version is 2.0.0.
+See the https://cortexmetrics.io/docs/getting-started/[Cortex documentation] for instructions.
+If your aim is to get up to speed quickly in a test environment, you can use Docker Compose to deploy a https://github.com/opennms-forge/stack-play/tree/master/standalone-cortex-minimal[minimal, standalone Cortex node].
 
 . Download the plugin's `.kar` file from GitHub (https://github.com/OpenNMS/opennms-cortex-tss-plugin/releases).
 . Copy the file into `$\{OPENNMS_HOME}/deploy`.
-. Enable and configure time series storage plugins in {page-component-title}:
+. Enable and configure time series storage plugins in {page-component-title}.
 +
-[source, console]
+.Create or modify the `$\{OPENNMS_HOME}/etc/opennms.properties.d/cortex.properties` file
+[source, properties]
 ----
-echo "org.opennms.timeseries.strategy=integration
+org.opennms.timeseries.strategy=integration
 org.opennms.timeseries.tin.metatags.tag.node=${node:label}
 org.opennms.timeseries.tin.metatags.tag.location=${node:location}
 org.opennms.timeseries.tin.metatags.tag.geohash=${node:geohash}
 org.opennms.timeseries.tin.metatags.tag.ifDescr=${interface:if-description}
-org.opennms.timeseries.tin.metatags.tag.label=${resource:label}" >> ${OPENNMS_HOME}/etc/opennms.properties.d/cortex.properties
+org.opennms.timeseries.tin.metatags.tag.label=${resource:label}
 ----
++
+NOTE: Make sure that the `org.opennms.timeseries.strategy` property is not set in `$\{OPENNMS_HOME}/etc/opennms.properties` or any other file within the `$\{OPENNMS_HOME}/etc/opennms.properties.d/` directory.
 
 . Configure the Cortex Time Series plugin in {page-component-title}.
 +
-Note that you can omit this step if you use the default settings.
-Substitute the correct host and port for your Cortex cluster in each place where cortex-service-host:9009 appears below.
+The `writeUrl` and `readUrl` properties are required.
+All other properties shown here are example values to overwrite the defaults.
+Keep an eye on your Cortex cluster performance and adjust as necessary.
 +
-[source, console]
+.Create or modify the `$\{OPENNMS_HOME}/etc/org.opennms.plugins.tss.cortex.cfg` file
+[source, properties]
 ----
-config:edit org.opennms.plugins.tss.cortex
-
-property-set writeUrl http://cortex-service-host:9009/api/prom/push
-property-set readUrl http://cortex-service-host:9009/prometheus/api/v1
-property-set maxConcurrentHttpConnections 100
-property-set writeTimeoutInMs 1000
-property-set readTimeoutInMs 1000
-property-set metricCacheSize 1000
-property-set externalTagsCacheSize 1000
-property-set bulkheadMaxWaitDurationInMs 9223372036854775807
-
-config:update
+writeUrl=http://cortex-service-host:9009/api/prom/push<1>
+readUrl=http://cortex-service-host:9009/prometheus/api/v1<2>
+maxConcurrentHttpConnections=100
+writeTimeoutInMs=1000
+readTimeoutInMs=1000
+metricCacheSize=1000
+externalTagsCacheSize=1000
+bulkheadMaxWaitDurationInMs=9223372036854775807
```

Comment:
why-not-both.gif

Reply:
![image](https://user-images.githubusercontent.com/26419758/192950242-64f869f9-cfbf-424c-ae25-13f7913e9697.png)